layouts/HomeContainer/HomeContainer.tsx

```diff
@@ -0,0 +1,18 @@
+"use client"
+
+import { MovieContextProvider } from "@/context/movieData.context";
+import { FC } from "react";
+
+interface HomeContainerTypeProp {
+    children: React.ReactNode
+}
+
+const HomeContainer:FC<HomeContainerTypeProp> = ({children}) => {
+    return (
```

Comment:
bug: Same here, the Prettier seems not configured correctly in your computer. [screenshot](https://github.com/TingChiehLin/top-movies/assets/14869087/f3b2bc32-9c05-4b15-91c8-5d4b27f0f151)